jekyll/_cci2/use-the-cli-and-vcs-for-config-policy-management.adoc

```diff
@@ -42,6 +59,22 @@ image:org-id.png[screenshot showing where to find your organization ID]
 
 Follow these steps to create a policy that checks the `version` of CircleCI config files to ensure it is greater than or equal to `2.1`.
 
+. Enable config policy management for your organization
```

Comment:
@sagar-connect I added enabling CPM for the org as the first step in the quickstart rather than a separate section. The options for this are described further down the doc too. Let me know if this seems ok

Reply:
Looks good to me.